Detectors/MUON/MCH/Workflow/src/PreClusterFinderSpec.cxx

```diff
@@ -102,9 +117,19 @@ class PreClusterFinderTask
     mPreClusters.reserve(nPreClusters); // to avoid reallocation if
     mUsedDigits.reserve(digits.size()); // the capacity is exceeded
     mPreClusterFinder.getPreClusters(mPreClusters, mUsedDigits);
-    if (mUsedDigits.size() != digits.size()) {
-      throw runtime_error("some digits have been lost during the preclustering");
-    }
+
+    // check sizes of input and output digits vectors
```

Comment:
hum, where's the actual check now ? (for the cases the check-no-leftover-digits option is not off)

Reply:
Ooops! I should not do coding on Friday afternoon ;-)

I have added the missing check, and combined all the commits into a single one for clarity.